internal/api/user/user.go

```diff
@@ -26,6 +26,7 @@ type Info struct {
 	// Groups are the group claims obtained from credentials that have been
 	// successfully verified by the server's authentication middleware.
 	Groups []string
+	Claims map[string]any
```

Comment:
Should be be removing the `Subject`, `Email`, and `Groups` fields?

Reply:
Yes, sorry, I missed them. I've removed them and fixed all the references.